environment-mac.yaml

```diff
@@ -28,7 +28,7 @@ dependencies:
     - kornia==0.6.0
     - -e git+https://github.com/openai/CLIP.git@main#egg=clip
     - -e git+https://github.com/CompVis/taming-transformers.git@master#egg=taming-transformers
-    - -e git+https://github.com/lstein/k-diffusion.git@master#egg=k-diffusion
```

Comment:
this isn't strictly required, so I can back out this change and it still remains functional for me on M1 if you feel strongly about keeping the fork within your control.

Reply:
Anyone know @Birch-san to ping them on whether this is a good idea? I don't know what their plans for this fork are.

Reply:
I originally forked the k-diffusion repository because it was missing something, either setup.py or requirements.txt, not sure which, and wouldn't install correctly. If this has been fixed I would be very happy to use a maintained fork, as I have no intention of maintaining k-diffusion on my own. The only caveat is that to test that the replacement fork works, you have to remove the installed k-diffusion package and reinstall it from the new fork. I would be paranoid and clean out the entire conda (or venv) environment and do the install from scratch.

Reply:
there are similar mitigations in my fork to ensure that it installs correctly (I forked from @hlky's fork, which fixed installing it as an egg from git).

yes, I think sensible to depend on my fork, or at least one with my MPS commits. They're not really upstream able to mainline k-diffusion because they make the code a bit less beautiful to work around bugs in PyTorch.

"maintained" is a strong word. 🙂 I make no promises. but certainly a good choice for now.

Reply:
I'm confused by @magnusviri 's narrative 'cause I'm not sure which are the "old" and "new" repos in this context. Can someone please confirm that when @Birch-san 's k-diffusion repo is referenced in environment.yaml we do not run into the conda hang? This is a blocking issue for me.

Reply:
Ok, I see confirmation that it is working later in the conversation. I'm going ahead with my review now.